src/CoreBundle/private/core/TModelBase.class.php

```diff
@@ -773,6 +773,15 @@ protected function getModuleObject($identifier, $field = 'id')
             $sViewPathReference = &$this->viewTemplate;
         } else {
             $oModule = $this->getModuleObject($this->aModuleConfig['model'], 'classname');
+            if (null === $oModule) {
```

Comment:
Is this another issue?

Reply:
Nope this is required here. When migrating to 6.3.x the sidebar module needs to be added to database by a migration script, but it is also rendered directly after login. So we need to avoid page explosions for this timespan.